File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/operation/delete/RecordsDeleteDao.java
package ru.citeck.ecos.records3.record.operation.delete;

import org.jetbrains.annotations.NotNull;
import ru.citeck.ecos.records3.source.dao.RecordsDao;

import java.util.List;

public interface RecordsDeleteDao extends RecordsDao {

    List<DelStatus> delete(@NotNull List<String> records);
}

File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/dao/delete/RecordsDeleteDao.kt
package ru.citeck.ecos.records3.record.dao.delete

import ru.citeck.ecos.records3.record.dao.RecordsDao
import kotlin.jvm.Throws

interface RecordsDeleteDao : RecordsDao {

    @Throws(Exception::class)
    fun delete(recordIds: List<String>): List<DelStatus>
}
